### DISCUSSION TITLE
Control Mesh Cleanup

### DISCUSSION THREAD

**Discussion by ramcdona** [elalish/manifold#557] [General]
Is there a way to control or turn off the mesh cleanup steps?

In my case, I don't want the original triangulation changed.  I.e. all original edges should remain when the operations are complete -- even if removing them results in the same geometric shape.

Here is the result I'm getting from Manifold:
![Screenshot 2023-09-11 at 10 46 40 AM](https://github.com/elalish/manifold/assets/1329600/93ab9edc-2317-42b7-95b9-ce4d0bc71494)

Here is the desired result from my current CSG engine (don't worry about the colors for now):
![Screenshot 2023-09-11 at 10 47 11 AM](https://github.com/elalish/manifold/assets/1329600/36c0c908-03db-44c1-8b99-472e31c8f963)

I need to maintain the original triangulation because it represents a computational mesh.  After processing through Manifold, I am going to feed this mesh to an analysis where the current Manifold result is unacceptable.

**Comment by pca006132** [comment]
https://github.com/elalish/manifold/discussions/547#discussioncomment-6879994

**Reply by ramcdona** [reply]
Thanks -- I have a bunch of properties that I intend to assign, so by the time I'm done with that, it will probably be taken care of.

**Reply by pca006132** [reply]
Note that verProperties is still relatively new and we have an issue about it: https://github.com/elalish/manifold/issues/536
